samples/YesSql.Samples.Web/Startup.cs

```diff
@@ -135,29 +135,28 @@ public async Task ConfigureAsync(IApplicationBuilder app, IWebHostEnvironment en
 
             var store = app.ApplicationServices.GetRequiredService<IStore>();
             store.RegisterIndexes(new[] { new BlogPostIndexProvider() });
-
-            using (var connection = store.Configuration.ConnectionFactory.CreateConnection())
+            Task.Run(async () =>
```

Comment:
Would need to be `awaited` (but in Configure() we can't use `await`) to not run in parrallel one task creating a table and another task using this table.

Reply:
@jtkech I added await on everything. This could maybe better placed in the controller or a new service like a migration

Reply:
@sebastienros I added the awaiter

Reply:
Better option if you are willing to, change the project to use the new model that doesn't use a Startup file and is all async.

Reply:
@sebastienros I changed it to using Program.